src/main/java/teammates/common/util/AdminLogQuery.java

```diff
@@ -0,0 +1,114 @@
+package teammates.common.util;
+
+import java.util.List;
+
+import com.google.appengine.api.log.LogQuery;
+import com.google.appengine.api.log.LogService.LogLevel;
+
+import teammates.common.util.Assumption;
+
+/**
+ * A wrapper class for LogQuery to retrieve logs from GAE server.
+ */
+public class AdminLogQuery {
+    /**
+     * A flag to decide whether to include application logs in result or not.
+     */
+    private static final boolean INCLUDE_APP_LOG = true;
+    
+    /**
+     * Affects the internal strategy to get logs. It doesn't affect the result.
+     */
+    private static final int BATCH_SIZE = 1000;
+    private static final LogLevel MIN_LOG_LEVEL = LogLevel.INFO;
+    
+    private LogQuery query;
+    private long startTime;
+    private long endTime;
+    
+    /**
+     * Sets values for query.
+     * If startTime is null, it will be considered as 0.
```

Comment:
The time used in LogQuery is Unix time. https://en.wikipedia.org/wiki/Unix_time
0 means it will take logs after Thursday, 1 January 1970 (since forever).